src/components/events/Events.js

```diff
@@ -193,7 +202,36 @@ const Events = () => {
         </Typography>
       </Box>
 
-      <Box sx={{ position: 'absolute', top: 150, left: '50%', transform: 'translateX(-50%)' }}>
+      <Box sx={{ width:'600px', position: 'absolute', top: 150, left: '50%', transform: 'translateX(-50%)', marginBottom: '20px' }}>
+
+        <Search
```

Comment:
We should consider adding these searches to the "My ..." (ex "My Events") as well for consistency